CRM/Contact/BAO/Query.php

```diff
@@ -4336,6 +4336,8 @@ public static function getQuery($params = NULL, $returnProperties = NULL, $count
    * @param string $apiEntity
    *   The api entity being called.
    *   This sort-of duplicates $mode in a confusing way. Probably not by design.
+   * @param bool $useGroupBy
+   *   Should the query be grouped by contact id?
```

Comment:
Sure, and if you really feel like it's the best description then I'm fine with it. My feeling is that you're describing what the parameter does; the question is already in the name `$useGroupBy`.

But you're right, looking over the code I can see (at least) 67 places in the core code where the parameter description ends in a question mark.